Research/rohanhh2/flutter_practice_pt3

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import 'package:random_string_generator/random_string_generator.dart';
+
+void main() => runApp(MyApp());
+
+// #docregion MyApp
+class MyApp extends StatelessWidget {
+  // #docregion build
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Startup Name Generator',
+      theme: ThemeData(
+        primaryColor: Colors.white,
+      ),
+      home: RandomWords(),
+    );
+  }
+  // #enddocregion build
+}
+// #enddocregion MyApp
+
+// #docregion RWS-var
+class _RandomWordsState extends State<RandomWords> {
+  final _suggestions = <WordPair>[];
+  final _saved = <WordPair>{};
+  final _biggerFont = TextStyle(fontSize: 18.0);
+  // #enddocregion RWS-var
+
+  // #docregion _buildSuggestions
+  Widget _buildSuggestions() {
+    return ListView.builder(
+        padding: EdgeInsets.all(16.0),
+        itemBuilder: /*1*/ (context, i) {
+          if (i.isOdd) return Divider(); /*2*/
+
+          final index = i ~/ 2; /*3*/
+          if (index >= _suggestions.length) {
+            _suggestions.addAll(generateWordPairs().take(10)); /*4*/
```

Comment:
generateWordPairs() is a method in the english_words package used in the tutorial